TRE-1992 add rate limiting documentation (#180)
MobilePayDev/MobilePayDev.github.io: docs/subscriptions/transition-to-one-platform.md

Some field names, like `mobile_phone_number`, will undergo changes; for instance, it will be referred to as `phoneNumber`. If clarity is needed, refer to [Recurring documentation](https://developer.vippsmobilepay.com/docs/APIs/recurring-api/) for field specifications.

:star: **Recommendation:** Avoid relying on specific values in `error_description.message` and `error_description.error_type`. Update your error handling processes to ensure flexibility in these two fields.

### 9.6 Rate limiting

> We have rate limiting in place for each endpoints of the facade. For simplicity reasons, we have put in place the same policies are the recurring-api. See [Rate limiting for recurring documentation](https://developer.vippsmobilepay.com/docs/APIs/recurring-api/recurring-api-guide/#rate-limiting) for details. 

Author: ```suggestion
We have rate limiting in place for each endpoints of the facade. For simplicity reasons, we have put in place the same policies as the recurring-api for most of the endpoints. 
```
